ci/openshiftio-appdev-docs/src/main/java/io/openshift/appdev/documentation/builder/HttpApplication.java

```diff
@@ -24,13 +24,15 @@
 public class HttpApplication extends AbstractVerticle {
 
    private static final String INDEX_PAGE = "index.html";
-
+   private static final String LAUNCHPAD_TEMPLATE_LATEST_URL = "https://raw.githubusercontent.com/openshiftio/launchpad-templates/v3/openshift/launchpad-template.yaml";
```

Comment:
Do we want to get this value from an external configuration (file, or whatever) ?